conf/logrotate.conf

```diff
@@ -20,6 +20,23 @@ notifempty
   endscript
 }
 
+## AUTH
+/var/log/st2/st2auth.log {
```

Comment:
I probably should have said use this "/var/log/st2/st2auth*.log" and then eliminate this identical section for audit:
/var/log/st2/st2auth.audit.log {
  daily
  rotate 5
  postrotate
    st2ctl reopen-log-files st2auth
  endscript
}